apiary-metastore-listener/README.md

```diff
@@ -53,6 +53,61 @@ The following shows an example JSON message representing an "INSERT" event:
       "fileChecksums":["123","456"],
       "partitionKeyValues":{"load_date":"2013-03-24","variant_code":"EN"}
     }
+    
+#### Alter Table
+The following shows an example JSON message representing an "ALTER_TABLE" event:
+
+    {
+      "protocolVersion":"1.0",
```

Comment:
Add space after `:` ? 
For eg, `"protocolVersion":"1.0"` --> `"protocolVersion": "1.0"`

Reply:
I am using the same formatting as "Create table" and "Insert" sections. Its a dump of the JSON which the module spits to SNS. I can change the formatting everywhere if everyone says.

Reply:
I think we either leave it as is or we put a space on either side like we've done in the Circus Train documentation for JSON (https://github.com/HotelsDotCom/circus-train/tree/master/circus-train-aws-sns#replication-start). I don't have a strong opinion either way.

Reply:
Alright. I have changed it to have spaces on both sides to make it consistent with circus-train.